src/main/resources/config/application-dev.properties

```diff
@@ -0,0 +1,30 @@
+# Logging level for the application loggers (all built-in routes and classes)
```

Comment:
Can we not have a common application.properties file, maybe `eip-openmrs-senaite.properties` that can be overriden for a particular environment?